cli/args/flags.rs

```diff
@@ -895,6 +895,49 @@ To evaluate code in the shell:
 
 /// Main entry point for parsing deno's command line flags.
 pub fn flags_from_vec(args: Vec<String>) -> clap::error::Result<Flags> {
+  // Fast path: "deno"
+  if args.len() < 2 {
+    let mut flags = Flags {
```

Comment:
Does flags also parse env vars? Need to make sure the clap flag parsing is actually pure

Reply:
I see three env vars in the flag parsing -- those will need to be dealt with.